lib/sessionClient.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const fetch = require('fetch-retry-or-die');
+const log = require('@financial-times/n-logger').default;
+const config = require('./helpers/config');
+const helpers = require('./helpers/helpers');
+const clientErrors = require('./clientErrors');
+
+const options = Object.assign({}, config.fetchOptions);
```

Comment:
Doh! That'll teach me to copy and paste a comment! I meant the `verify()` function :)